src/org/jitsi/sctp4j/Sctp.java

```diff
@@ -174,6 +174,7 @@ public static synchronized void init()
         //    return;
         if(!initialized)
         {
+            logger.error("Init'ing brian's patched usrsctp");
```

Comment:
haha actually this is intentional :) i can remove the 'brian' part of it if we want, but i added some prints so we can know that the proper lib is loaded